kili/mutations/issue/__init__.py

```diff
@@ -0,0 +1,85 @@
+"""
+Issue mutations
+"""
+
+from typing import Literal, Optional
+from typeguard import typechecked
+from .helpers import get_issue_number
+from ...helpers import Compatible, format_result
+from .queries import GQL_APPEND_TO_ISSUES
+from ..comment.queries import GQL_APPEND_TO_COMMENTS
+from ...queries.label import QueriesLabel
+
+
+class MutationsIssue:
+    """Set of Issue mutations."""
+
+    # pylint: disable=too-few-public-methods,too-many-arguments,too-many-locals
+
+    def __init__(self, auth):
+        """Initialize the subclass.
+
+        Args:
+            auth: KiliAuth object
+        """
+        self.auth = auth
+
+    @Compatible(["v1", "v2"])
```

Comment:
We should remove this decorator, it's one year old and not useful anymore

Reply:
@mDuval1 Ok, what was this feature for ?
We can completely delete all the `Compatible` class and all its logic ?